File: 백준/Bronze/5358. Football Team/Football Team.py
import sys
input = sys.stdin.readline

while (1):
    s = input().rstrip().replace('i', '*').replace('I', '_').replace('e', 'i').replace('E', 'I').replace('*', 'e').replace('_', 'E')
    if s == '':
        break
    print(s)

File: 백준/Bronze/5358. Football Team/Football Team.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String s;

        while ((s = br.readLine()) != null) {
            s = s.replace('i', '*');
            s = s.replace('I', '_');
            s = s.replace('e', 'i');
            s = s.replace('E', 'I');
            s = s.replace('*', 'e');
            s = s.replace('_', 'E');

            System.out.println(s);
        }
    }
}